packages/flutter/lib/src/widgets/snapshot_widget.dart

```diff
@@ -368,7 +377,7 @@ abstract class SnapshotPainter extends ChangeNotifier  {
   /// }
   /// ```
   /// {@end-tool}
-  void paintSnapshot(PaintingContext context, Offset offset, Size size, ui.Image image, double pixelRatio);
+  void paintSnapshot(PaintingContext context, Offset offset, Size size, ui.Image image, Size imageSize, double pixelRatio);
```

Comment:
Please update the docs to include the new parameter and how to use it.

Reply:
done